docs/blockchain/contracts/nft-contract/nft-actions/stofrcfg.a.experimental.md

```diff
@@ -0,0 +1,70 @@
+---
+title: 'stofrcfg.a'
+order: 36
+
+---
+
+# stofrcfg.a
+
+Set global uniq offer configuration
```

Comment:
we can use either "uniq offer" or "buy offer" in action pages, but should  not use both.

Reply:
keep as `uniq offer`